src/main/java/com/sublinks/sublinksapi/api/sublinks/v1/person/mappers/PersonMapper.java

```diff
@@ -0,0 +1,58 @@
+package com.sublinks.sublinksapi.api.sublinks.v1.person.mappers;
+
+import com.sublinks.sublinksapi.api.lemmy.v3.utils.DateUtils;
```

Comment:
Try to avoid cross-dependencies between Lemmy API and Sublinks API. There is a possibility that we'll drop Lemmy API support at some point.